pkg/bindings/test/images_test.go

```diff
@@ -409,7 +409,8 @@ var _ = Describe("Podman images", func() {
 		results, err := images.Build(bt.conn, []string{"fixture/Containerfile"}, entities.BuildOptions{})
 		Expect(err).ToNot(HaveOccurred())
 		Expect(*results).To(MatchFields(IgnoreMissing, Fields{
-			"ID": Not(BeEmpty()),
+			"ID":         Not(BeEmpty()),
+			"SaveFormat": BeEmpty(),
```

Comment:
Would it be better to fill this field in at [the end of pkg/bindings/images.Build()](https://github.com/containers/podman/blob/fe828d223812ddb5aa2525911108a7d276e062af/pkg/bindings/images/build.go#L633) and test for that?